src/utils/get-core-member-logins.ts

```diff
@@ -14,14 +14,13 @@ limitations under the License.
 import * as core from '@actions/core';
 import { CodeOwnersEntry, loadOwners, matchFile } from 'codeowners-utils';
 import { uniq, union } from 'lodash';
-import { context } from '@actions/github';
 import { getChangedFilepaths } from './get-changed-filepaths';
 import { map } from 'bluebird';
-import { octokit } from '../octokit';
 import { convertToTeamSlug } from './convert-to-team-slug';
+import { paginateMembersInOrg } from './paginate-members-in-org';
 
-export const getCoreMemberLogins = async (pull_number: number, teams?: string[]) => {
-  const codeOwners = teams ?? getCodeOwnersFromEntries(await getRequiredCodeOwnersEntries(pull_number));
+export const getCoreMemberLogins = async (pull_number: number, teams?: string[], codeowners_overrides?: string) => {
+  const codeOwners = teams ?? getCodeOwnersFromEntries(await getRequiredCodeOwnersEntries(pull_number, codeowners_overrides));
```

Comment:
`getRequiredCodeOwnersEntries` already supports `codeowners_overrides` thanks to #706, just need to pass forward the argument